CoherentLabs/GameUIComponents
# Megapackage

Added one package that depends on all components allowing you to install them all with one command.
Added readme.

Test Plan:
1. Checkout megapackage-demo branch
2. Run `npm run build:dev`
3. Navigate to `/coherent-gameface-ui`
4. To link dependencies from the local file system - run 
```
npm link coherent-gameface-automatic-grid coherent-gameface-components coherent-gameface-checkbox coherent-gameface-dropdown coherent-gameface-form-control coherent-gameface-grid coherent-gameface-menu coherent-gameface-modal coherent-gameface-progress-bar coherent-gameface-radial-menu coherent-gameface-radio-button coherent-gameface-rangeslider coherent-gameface-router coherent-gameface-scrollable-container coherent-gameface-slider coherent-gameface-switch coherent-gameface-stepper coherent-gameface-tabs coherent-gameface-tooltip coherent-gameface-text-field coherent-gameface-accordion-menu coherent-gameface-interaction-manager
```

5. Run `npm link` - to create a global link to `coherent-gameface-ui`
6. Go to `/coherent-gameface-ui/demo/project-that-uses-the-suite` and run
`npm link coherent-gameface-ui`
7. Go to `tools/cli` and run `npm i` and then `npm link`
8. Go to `/coherent-gameface-ui/demo/project-that-uses-the-suite` and run `coherent-guic-cli build:demo`
9. Open `/coherent-gameface-ui/demo/project-that-uses-the-suite/demo/demo.html` - you should see a page with multiple working components:
![megabundle](https://github.com/CoherentLabs/GameUIComponents/assets/19714551/dd8804cb-845f-4161-85d4-cd74fd088798)
